sdk/Sdk.Generators/FunctionMetadataProviderGenerator/FunctionMetadataProviderGenerator.Parser.cs

```diff
@@ -481,35 +481,47 @@ private bool TryGetReturnTypePropertyBindings(ITypeSymbol returnTypeSymbol, bool
                         foundHttpOutput = true;
                         bindingsList.Add(GetHttpReturnBinding(prop.Name));
                     }
-                    else if (prop.GetAttributes().Length > 0) // check if this property has any attributes
+
+                    if (prop.GetAttributes().Length > 0)
                     {
-                        var foundPropertyOutputAttr = false;
+                        var bindingAttributes = prop.GetAttributes().Where(p => p.AttributeClass!.IsOrDerivedFrom(_knownFunctionMetadataTypes.BindingAttribute));
+
+                        if (bindingAttributes.Count() > 1)
+                        {
+                            _context.ReportDiagnostic(Diagnostic.Create(DiagnosticDescriptors.MultipleBindingsGroupedTogether, Location.None, new object[] { "Property", prop.Name.ToString() }));
```

Comment:
Can you remind me what code can cause this to diagnostic to be triggered? (487 to 491) ? What attribute customer may write (which inherits from bindingattribute) ?

Reply:
[Here](https://github.com/Azure/azure-functions-dotnet-worker/blob/c1b2182e4611609e9b517828e871369a714e96b2/test/FunctionMetadataGeneratorTests/FunctionMetadataGeneratorTests.cs#L1362) is an example of stacking output bindings attributes from legacy tests. The check here is in case a user stacks multiple binding attributes on a property in the return type (ex. trying to bind one prop as a queue output and a blob output).